code/game/objects/structures/transit_tubes/transit_tube_construction.dm

```diff
@@ -35,6 +35,13 @@
 	to_chat(user, "<span class='notice'>You start attaching the [name]...</span>")
 	add_fingerprint(user)
 	if(I.use_tool(src, user, time_to_unwrench, volume=50))
+		var/turf/source_turf = get_turf(loc)
+		var/existing_tubes = 0
+		for(var/obj/structure/transit_tube/tube in source_turf)
+			existing_tubes +=1
+			if(existing_tubes >= 2)
+				to_chat(user, "<span class='warning'>You cannot wrench anymore transit tubes!</span> ")

```

Comment:
do this prior to the `use_tool` call

Reply:
If I place it before use_tool I can still wrench many tubes before the first tubes finished wrenching.

Reply:
Hm. I really don't care for the fact that this will only happen after the timer, resulting in the completion sound playing, and the user having to sit through the delay to even get to the warning.

Roll this into a seperate proc, check it at the start of wrench_act, and (I think) you could possible have it be an extra check under use_tool (extra_checks param), that way it'll fail if the condition is met during wrenching.

Reply:
Since use_tool will call the check anyway whats the use in putting it at the start of wrench_act?

Reply:
Done, should be good now.